price-feeder/oracle/util.go

```diff
@@ -2,48 +2,114 @@ package oracle
 
 import (
 	"fmt"
+	"sort"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/umee-network/umee/price-feeder/oracle/provider"
 )
 
+var minimumTimeWeight = sdk.MustNewDecFromStr("0.2")
+
+// tvwapCandlePeriod represents the time period we use for tvwap in minutes.
+const tvwapCandlePeriod = 3
```

Comment:
So all providers only use `tvwapCandlePeriod` last traded prices?

Reply:
Tvwap only uses prices that were reported after `tvwapCandlePeriod` (3 minutes prior) when calculating, whereas providers store prices that were reported after `providerCandlePeriod` (10 minutes prior).